src/main/java/com/solvd/micro9/tickets/messaging/KfConsumer.java

```diff
@@ -0,0 +1,28 @@
+package com.solvd.micro9.tickets.messaging;
+
+import com.solvd.micro9.tickets.service.TicketService;
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.kafka.core.reactive.ReactiveKafkaConsumerTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class KfConsumer {
+
+    private final ReactiveKafkaConsumerTemplate<String, Long> receiver;
+    private final TicketService ticketService;
+
+    @PostConstruct
+    public void fetch() {
```

Comment:
try to declare methods only in interfaces

Reply:
you mean creating KfConsumer interface and then its implementation?